### PR TITLE
Create application

### PR DESCRIPTION
**Pull request by dfarr**
Hello OSCON!!!

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/reviewninja/oscon-2016/pull/1" target="_blank">ReviewNinja

File: app.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var http = require('http');
+
+var server = http.createServer(function(req, res) {
+  
+   response.writeHead(200, {"Content-Type": "text/plain"});
```
Comment (casche) [inline app.js]: !fix hey david, you need to align the whitespace here! thanks man.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  response.end("Hello OSCON!!!\n");
+  
+}
```
Comment (casche) [inline app.js]: !fix you should put the }) on the same line!

File: app.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var http = require('http');
+
+var server = http.createServer(function(req, res) {
+  
+   response.writeHead(200, {"Content-Type": "text/plain"});
+  response.end("Hello OSCON!!!\n");
```
Comment (casche) [inline app.js]: !fix I think the proper variable name is 'res' and not 'response'

File: app.js (hunk 2 above)
Comment (dfarr) [inline app.js]: !fixed

File: app.js (hunk 1 above)
Comment (dfarr) [inline app.js]: !resolved

File: app.js (hunk 3 above)
Comment (dfarr) [inline app.js]: !resolved

**Comment by casche** [comment]
Looks good @dfarr

**Pull request merged**